app/models/practice.rb

```diff
@@ -111,6 +111,7 @@ def origin_picture_s3_presigned_url(style = nil)
 
   scope :published,   -> { where(published: true) }
   scope :unpublished,  -> { where(published: false) }
+  scope :Get_Practice_Owner_Emails, -> {where(user_id: exists?)}
```

Comment:
the ruby way: let's lower case this

Reply:
good catch... I didn't think Ruby was smart enough to upper case the text for the button display.. :)